File: packages/feishu/src/feishuScopes.ts
export const feishuCalendarScopes = [
  {
    key: "calendar:calendar",
    label: "更新日历和日程信息"
  },
  {
    key: "calendar:calendar:readonly",
    label: "读取日历、日程和忙闲信息"
  },
  {
    key: "calendar:calendar:read",
    label: "读取日历信息"
  },
  {
    key: "calendar:calendar.event:read",
    label: "读取日程列表"
  },
  {
    key: "calendar:calendar.event:create",
    label: "创建日程"
  },
  {
    key: "calendar:calendar.event:update",
    label: "更新日程"
  },
  {
    key: "calendar:calendar.event:delete",
    label: "删除日程"
  },
  {
    key: "contact:user.email:readonly",
    label: "读取参与人邮箱（可选，未默认申请）"
  },
] as const;

export type FeishuScopeKey = (typeof feishuCalendarScopes)[number]["key"];

File: apps/android/app/src/main/java/ai/tiiny/resolve/FeishuScopes.kt
package ai.tiiny.resolve

data class FeishuScope(val key: String, val label: String)

val FeishuCalendarScopes = listOf(
    FeishuScope("offline_access", "离线访问已授权数据"),
    FeishuScope("calendar:calendar:readonly", "读取主日历"),
    FeishuScope("calendar:calendar", "管理日历"),
    FeishuScope("calendar:event:readonly", "读取日程列表"),
    FeishuScope("calendar:event", "创建、更新、删除日程"),
    FeishuScope("contact:user.email:readonly", "读取参与人邮箱（可选）")
)
